neuraxle/steps/loop.py

```diff
@@ -21,11 +21,11 @@
 import copy
 from typing import List, Any
 
-from neuraxle.base import MetaStepMixin, BaseStep
+from neuraxle.base import MetaStepMixin, BaseStep, DataContainer, ExecutionContext, ListDataContainer, ForceHandleMixin
 from neuraxle.hyperparams.space import HyperparameterSamples, HyperparameterSpace
 
 
-class ForEachDataInput(MetaStepMixin, BaseStep):
+class ForEachDataInput(ForceHandleMixin, MetaStepMixin, BaseStep):
```

Comment:
It's the second time I ask that the `ForEachDataInput` doesn't inherit from the `ForceHandleMixin`. It's for the following reasons: 
- The `ForEachDataInput` can fit or transform or fit_transform like all other objects. 
- The `ForEachDataInput` doesn't need to be executed ONLY in pipelines. It is a normal step after all. 
- The `ForEachDataInput` is already used in other project and it will break if we make it impossible to fit or transform. 

Conclusion: 
- Use the `ForceHandleMixin` ONLY in objects that MUST be used within pipelines. It is because the `ForceHandleMixin` not only forces object to use handles, but also it is negative as it forbids them to have normal behaviors. 

Suggestion: 
- Rename `ForceHandleMixin` to `ForceAlwaysHandleMixin`. 
- The `ForEachDataInput` doesn't need to inherit from `ForceAlwaysHandleMixin`. 
- Don't remove `fit` and `transform` from `ForEachDataInput`. You only needed to add behavior to this class, not remove it.